atlasdb-cassandra/src/main/java/com/palantir/atlasdb/cassandra/CassandraKeyValueServiceConfig.java

```diff
@@ -124,15 +123,12 @@ default int poolRefreshIntervalSeconds() {
     /**
      * The minimal period we wait to check if a Cassandra node is healthy after it has been blacklisted.
      *
-     * @deprecated Use {@link CassandraKeyValueServiceRuntimeConfig#unresponsiveHostBackoffTimeSeconds()} to make this
+     * @deprecated Use {@link CassandraKeyValueServiceRuntimeConfig#unresponsiveHostBackoffTimeSeconds()} to make
+     * this
      * value live-reloadable.
```

Comment:
sketchily formatted javadoc

Reply:
Odd, I saw this, looked at my own code and it wasn't there (with no committable changes or unpushed commits) - initially presumed it was GH diff being weird. Played around with git to show the change, so should be fixed now.